http-api/Cargo.toml

```diff
@@ -23,6 +24,8 @@ either = { version = "1.6" }
 tracing = "0.1"
 tracing-subscriber = "0.2"
 async-trait = "0.1"
+ethers-core = "0.6.3"
+rand = "0.8.5"
```

Comment:
If you just need a simple rng, you can use `fastrand` which doesn't pull in a ton of dependencies.